book/translation.rst

```diff
@@ -502,6 +502,11 @@ as the locale for the current request.
 You can now use the locale to create routes to other translated pages
 in your application.
 
+.. tip::
+
+    Read :doc:`/cookbook/routing/service_container_parameters` to learn, how to
+    avoid hardcoding the ``_locale`` requirement in all your routes.
```

Comment:
I took it from the referenced entry 
> You could hardcode your `_locale` requirement in all your routes [...]

For me it's referring to the route option but I understand your question, it is maybe also interpretable as the regular word. As a non-native speaker I would say, the sentence is still valid then, isn't it?